src/components/inspector/options-list.vue

```diff
@@ -332,27 +331,27 @@ export default {
     this.allowMultiSelect = this.options.allowMultiSelect;
   },
   methods: {
-    jsonDataChange() {	
-      let jsonList = [];	
-      try {	
-        jsonList = JSON.parse(this.jsonData);	
-        if (jsonList.constructor !== Array && jsonList.constructor !== Object) {	
-          throw Error('String does not represent a valid JSON');	
-        }	
-      }	
-      catch (err) {	
-        this.jsonError = err.message;	
-        return;	
+    jsonDataChange() {
+      let jsonList = [];
+      try {
+        jsonList = JSON.parse(this.jsonData);
+        if (jsonList.constructor !== Array && jsonList.constructor !== Object) {
+          throw Error('String does not represent a valid JSON');
+        }
       }	
-      this.optionsList = [];	
-      const that = this;	
-      jsonList.forEach (item => {	
-        that.optionsList.push({	
-          [that.keyField] : item[that.keyField],	
-          [that.valueField] : item[that.valueField],	
-        });	
-      });	
-      this.jsonError = '';	
+      catch (err) {
+        this.jsonError = err.message;
+        return;
+      }
+      this.optionsList = [];
+      const that = this;
+      jsonList.forEach (item => {
+        that.optionsList.push({
+          [that.keyField] : item[that.keyField],
+          [that.valueField] : item[that.valueField],
```

Comment:
Isn't the `that=this` business unneeded in ES6? Also couldn't this be done using a map which would simplify it, something like:
```suggestion
     this.optionsList = jsonList.map(item => {
         return {
             [this.keyField] : item[this.keyField],
             [this.valueField] : item[this.valueField]
     });
```